flink-table/flink-sql-client/src/test/resources/test-sql-client-catalogs.yaml

```diff
@@ -28,49 +28,22 @@ tables:
     type: source-table
     $VAR_UPDATE_MODE
     schema:
-      - name: IntegerField1
-        type: INT
-      - name: StringField1
+      - name: dec
+        type: DECIMAL
+      - name: vch
         type: VARCHAR
     connector:
       type: filesystem
       path: "$VAR_SOURCE_PATH1"
     format:
       type: csv
       fields:
-        - name: IntegerField1
-          type: INT
-        - name: StringField1
+        - name: dec
+          type: DECIMAL
+        - name: vch
           type: VARCHAR
       line-delimiter: "\n"
       comment-prefix: "#"
-  - name: TestView1
```

Comment:
why deleting the following section?

Reply:
Those tables/views are not used. I will revert these changes if you prefer to keep them.

Reply:
Yeah, let's revert them.

They are not used in queries, but are registered when sql cli bootstraps to test registration logic.